2.x-activity-module/mod/aspirelists/launch.php

```diff
@@ -20,5 +20,11 @@
 
 $list->cmid = $cm->id;
 aspirelists_add_lti_properties($list);
-lti_view($list);
-
+if($CFG->version >= 2015111600) {
```

Comment:
I've forgotten some of the Moodle release details, but what happens if a security fix of 2.x comes out?  Will its version number not be 2016*?